client/modules/User/components/DashboardTabSwitcher.jsx

```diff
@@ -28,12 +28,14 @@ Tab.propTypes = {
   to: PropTypes.string.isRequired,
 };
 
-const DashboardTabSwitcher = ({ currentTab, isOwner, username }) => (
+const DashboardTabSwitcher = ({
```

Comment:
I think where a component is declared in it's own file, and so can be imported anywhere in the app, it makes sense to use `withTranslation()` or in this case, the [`useTranslation()` hook](https://react.i18next.com/latest/usetranslation-hook).

This means the parent component doesn't have to pass in the `t` prop.

Reply:
Also, why is the translation function called `translate` rather than `t`? Does it perform a different function?

Reply:
I'm thinking in exposing the translated component in this way

const DashboardTabSwitcherPublic = withTranslation()(DashboardTabSwitcher);
export { DashboardTabSwitcherPublic as default, TabKey };

Reply:
This looks like a good approach.

You could also use the [`useTranslation()` hook](https://react.i18next.com/latest/usetranslation-hook).